### ISSUE TITLE
Squeezr suddenly stopped working...

### ISSUE THREAD

**Issue by rhythmstixx** [jkphl/squeezr#17]
Hi,
Our Squeezr implementation suddenly stopped working after 2 months of running smoothly. Interestingly, it stopped on all sites (dev, test, production) at the same time.

No folders are created, and no cache images or symlinks either. Before everything worked perfectly. 
Could you get in touch and help out? I appreciate your help.

Jan

**Comment by rhythmstixx** [comment]
I got it to work locally again, but not on TEST or PRODUCTION. The cookies are properly set, but the images are not being rendered and no symlinks created either. 

Below is a copy of our .htaccess file:

`# Use PHP5 Single php.ini as default
AddHandler application/x-httpd-phpbetas .php

#Options +FollowSymlinks
Options +SymLinksIfOwnerMatch

RewriteEngine on
RewriteBase /

RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule ^media/(.*)$ /uploads/$2 [R=301,L]

# Redirect non-existing image cache requests to squeezr
RewriteCond %{REQUEST_URI} !core
RewriteCond %{REQUEST_URI} !plugins
RewriteCond %{REQUEST_URI} !themes
RewriteCond %{REQUEST_URI} !wp
RewriteCond %{REQUEST_FILENAME} -f
RewriteCond %{ENV:REDIRECT_BREAKPOINT} !\d+px
RewriteCond %{QUERY_STRING} !^([^&]*&)*squeezr=(0|false|no)
RewriteCond %{HTTP_COOKIE} squeezr.images=(\d+px) [NC]
RewriteRule ^(.+)(\.(?:jpe?g|gif|png))$ squeezr/cache/$1-%1$2 [NC,E=BREAKPOINT:%1,L]

RewriteCond %{HTTP_HOST} !^paytonbinnings\.com$
RewriteRule ^/?robots\.txt$ /devbots.txt [L,R=301]

RewriteRule ^properties(\/)$ /our-properties [R=301,L]
RewriteRule ^index\.php$ - [L]

RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule . /index.php [L]`

Thank you for your assistance!

**Comment by jkphl** [comment]
I'm very sorry but I can hardly help you debugging your very specific setup. As far as I see you're using Wordpress, right? You don't happen to have updated to 4.6 recently? Any chances Wordpress itself does intercept the image requests? What did you do to get it running locally again?

**Comment by rhythmstixx** [comment]
Hi Joschi,

Thank you for your feedback - I appreciate it!
I finally got it back to work on the test and production servers again. Turns out the hosting provider had turned on default caching for all the domains without informing me, and therefore Squeezr was never run anymore. I have a cron job set up to clear the Squeezr cache once a week, so once the cache was cleared the pages always had to load the original images - which are huge (5+MB each). Once I turned the cache off, it all worked smoothly again. Not the first place I would look, so I kept thinking something was suddenly wrong with the .htaccess files.

Thanks again,

Jan
